backend/src/services/aws.ts

```diff
@@ -150,7 +150,7 @@ export async function detectSentimentBatch(textArray) {
         negative,
         neutral: 100 * i.SentimentScore.Neutral,
         mixed: 100 * i.SentimentScore.Mixed,
-        sentiment: Math.round(100 * ((positive - negative) / (2 * (positive + negative)) + 0.5)),
```

Comment:
Is this supposed to be changed in this PR?

Reply:
Not related to the PR. I found it was wrong and changed it.